src/components/layout/Section.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+
+export const VALID_BG_NORMAL = ['white', 'haze', 'lightGray'];
+
+export const VALID_BG_INVERTED = ['navy'];
+
+export const VALID_PADDING = ['default', 'double', 'half', 'none'];
+
+/**
+ * @param {String} axis 'h' or 'v'
+ * @param {String} amount
+ * @returns {String} padding utility classes
+ */
+export const getPaddingClasses = (axis, amount) => {
+  const sides = axis === 'h' ? ['left', 'right'] : ['top', 'bottom'];
+
+  let classes = `padding--${sides[0]} padding--${sides[1]}`;
+  if (amount !== 'default') {
+    classes = `padding--${sides[0]}--${amount} padding--${sides[1]}--${amount}`;
+  }
+
+  return classes;
+};
+
+/**
+ * @param {String} direction
+ * @returns {String} border utility classes
+ */
+export const getBorderClasses = (direction) => {
+  let classes = '';
+  switch (direction) {
+    case 'h':
+      classes = 'border--left border--right';
+      break;
+    case 'v':
+      classes = 'border--top border--bottom';
+      break;
+    default:
+      classes = `border--${direction}`;
+  }
+  return classes;
+};
+
+/**
+ * @param {Object} props react props
+ * @returns {ReactElement}
+ */
+const Section = ({ hPadding, vPadding, bgColor, border, children }) => {
+  const classNames = cx(
+    `bgColor--${bgColor}`,
+    'display--block',
+    getPaddingClasses('h', hPadding),
+    getPaddingClasses('v', vPadding),
+    {
+      inverted: VALID_BG_INVERTED.includes(bgColor),
+      [getBorderClasses(border)]: Boolean(border),
+    }
+  );
+  const styles = {
+    backgroundClip: 'border-box',
+  };
```

Comment:
Should we make a CSS Module for this, just so no one gets any funny ideas? (or, reusable class)

Reply:
yeah, can't hurt